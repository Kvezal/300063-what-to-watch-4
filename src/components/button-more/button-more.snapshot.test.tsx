import * as React from "react";
import * as render from "react-test-renderer";

import ButtonMore from "./button-more";


test(`should render component`, () => {
  const tree = render
    .create(
        <ButtonMore
          hide={false}
          onButtonClick={() => null}
        >Test</ButtonMore>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
