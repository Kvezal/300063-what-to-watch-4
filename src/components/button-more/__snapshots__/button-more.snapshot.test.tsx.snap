// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<button
  className="catalog__button"
  onClick={[Function]}
  type="button"
>
  Test
</button>
`;
